modules/qualitycontrol/CodecSelection.spec.js

```diff
@@ -239,11 +270,14 @@ describe('Codec Selection', () => {
 
             participant1 = new MockParticipant('remote-1');
             conference.addParticipant(participant1, [ 'av1', 'vp9', 'vp8' ]);
+
+            await nextTick(1000);
+            expect(jingleSession.setVideoCodecs).toHaveBeenCalledTimes(1);
             expect(jingleSession.setVideoCodecs).toHaveBeenCalledWith([ 'av1', 'vp9', 'vp8' ], undefined);
 
             participant2 = new MockParticipant('remote-2');
             conference.addParticipant(participant2, [ 'av1', 'vp9', 'vp8' ]);
-            expect(jingleSession.setVideoCodecs).toHaveBeenCalledWith([ 'av1', 'vp9', 'vp8' ], undefined);
+            expect(jingleSession.setVideoCodecs).toHaveBeenCalledTimes(1);
```

Comment:
Don't we also want to know that the codec selection ended up as expected?

Reply:
I think I removed this unintentionally, will add it back.

Reply:
👍